src/components/modal/UALBox.js

```diff
@@ -4,7 +4,7 @@ import PropTypes from 'prop-types'
 import i18n from '../../i18n'
 
 import { UALContainer } from './UALContainer'
-import { withUAL } from '../provider/UALContext'
+import { withUAL } from '../provider/withUAL'
```

Comment:
To stay consistent, should this be `'../provider/WithUAL'`?

Reply:
I think in this case since its only export is `withUAL`, which is not a class, I want to keep this file lowercase `withUAL` to match